region/lowernorfair/east.json

```diff
@@ -960,7 +967,36 @@
           "to": [
             {
               "id": 3,
-              "requires": [ "canPassBombPassages" ],
+              "requires": [
+                {"or": [
+                  {"and": [
+                    "heatProof",
+                    {"or": [
+                      "canUseMorphBombs",
+                      {"and": [
+                        "canUsePowerBombs",
+                        {"cost": [
+                          {"ammo": [{
+                            "type": "PowerBomb",
+                            "count": 3
+                          }]}
+                        ]}
+                      ]}
+                    ]}
+                  ]},
+                  {"and": [
+                    "canHeatRun",
+                    "canUsePowerBombs",
```

Comment:
Is there a reason morph bombs here are kept out of heatRun logic? It feels like it should only add a few hundred frames.

Reply:
From what I gather if you're not fast to get through the morph ball passage the Kihunter above becomes hell to handle.

Reply:
Hmm. Yeah he seems to like moving down to the door level. However, he doesn't seem to actually stick to your passage up so if you want an extra second or two it seems quite doable to avoid him with morph. He actually seems to like going back up.

Reply:
You're right if you wait a little while longer he seems to go away. Adding bombs with that in mind.